services/src/main/scala/cromwell/pipeline/service/UserService.scala

```diff
@@ -31,30 +33,56 @@ trait UserService {
 
 object UserService {
 
+  object Exceptions {
+    sealed abstract class UserServiceException(message: String) extends ServiceException(message)
+
+    final case class AccessDenied(message: String = "Access denied") extends UserServiceException(message)
+    final case class NotFound(message: String = "User with this id doesn't exist") extends UserServiceException(message)
+    final case class WrongEnteredPassword(message: String = "Users password differs from entered")
```

Comment:
```suggestion
    final case class WrongPassword(message: String = "Users password differs from entered")
```
I think `WrongRepeatedPassword` and `WrongEnteredPassword` are basically the same thing so you can just use different error messages for them.

Reply:
Yep, that's true. I got it.
Done.